frame/executive/src/lib.rs

```diff
@@ -566,6 +639,14 @@ where
 	/// This doesn't attempt to validate anything regarding the block, but it builds a list of uxt
 	/// hashes.
 	pub fn apply_extrinsic(uxt: Block::Extrinsic) -> ApplyExtrinsicResult {
+		Self::apply_extrinsic_with_mode(uxt, ExtrinsicInclusionModeQuery::get())
+	}
+
+	/// Same as `apply_extrinsic` but gets the `mode` directly passed in.
+	pub fn apply_extrinsic_with_mode(
```

Comment:
```suggestion
	fn apply_extrinsic_with_mode(
```

might help the readability of this file if we only make the things used in the runtime public, and even name them such that it is clear in which API they are meant to be used eg `pub fn block_builder_finalze_block()`

Reply:
I will make them private otherwise, yes. Not sure if we should do the renaming though. WDYT @bkchr?